Added member history

V24039 opened the pull request:
## PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)
- [ ] - Unit and e2e tests for the changes that have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix (fixes an issue)
- [x] Feature (adds functionality)
- [ ] Code style update
- [ ] Refactoring (no functional changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation changes
- [ ] Other... Please describe:

## What is the current behavior?

## What is the new behavior?
Added member history section and gap fix for User Statistics
![Screenshot 2025-04-27 194231](https://github.com/user-attachments/assets/06787c50-71f0-4d7c-b804-8c3d8b87538a)
![Screenshot 2025-04-27 194303](https://github.com/user-attachments/assets/a0dba42e-66d8-45ee-9987-0765617fb699)

_Describe the new behaviour_
_If useful, provide screenshot or capture to highlight main changes_

## Does this PR introduce a DB Schema Change or Migration?

- [ ] Yes
- [x] No

## Git Issues

Closes #4098 (2)

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of our monthly maintainers call (first Monday of the month)

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on Discord in the [Community Platform `development` channel](https://discord.com/channels/586676777334865928/938781727017558018).

V24039 commented:
Hi @dalibormrska thankyou for reviewing the PR. So for it has good, I got learn new things while going through code, I found it bit difficult going through the codebase as it is large😅😅, but as it well structured and also the file names were added in the issue desp it helped a lot.
I will go through the comments and make the changes required. Regarding the divider line, it is adjusted according to parent component, the flex stretch makes it cover the whole component.

benfurber commented:
Hey @V24039, thanks for your patience - I was on holiday last week. 

1. As everything in `src/pages/User/content/MemberHistory.tsx` is styling, it should be in the component library please.
2. From @dalibormrska's feedback I think the gap and divider thickness relate to this PR and maybe the rest to others?

davehakkens commented:
Looking nice @V24039.
Curious to see this going live and see the dates on the profiles :)

Does anyone know where the "last active" data is based upon?
- Last login date?

V24039 commented:
Hi @benfurber, @dalibormrska Sorry for the delay. I moved the file to components folder and added a test file for it. Worked on thickness of divider and increased the gap to 20px

davehakkens commented:
@benfurber could we also release this behind authwrapper?
Its not tested with live data + unclear what 'last active' means and how its effected with supabase migration. 
So i could see some dates not making sense.
This gives us some time to safely browser around and check profiles and edge cases
